src/net/sourceforge/kolmafia/persistence/FaxBotDatabase.java

```diff
@@ -104,7 +104,9 @@ private static void configureFaxBot(final BotData data) {
     }
   }
 
-  public static final FaxBot getFaxbot(final int i) {
+  public static final FaxBot getFaxbot(int i) {
+    i = i % faxbots.size();
```

Comment:
I don't know why this was ever allowed to return null - perhaps if there were no faxbots defined in the file? - but the modulo operation guarantees that the index will be less than the size so the return statement can be simplified to take advantage of that.